cdap-ui/app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.js

```diff
@@ -231,7 +237,7 @@ function _filterPipelineList(
     });
 
   let pipelineList = appsList.filter((app) => {
-    let isPipeline = app.artifact.name === DATA_PIPELINE_ARTIFACT;
+    let isPipeline = GLOBALS.programType[app.artifact.name] === WORKFLOW_TYPE;
```

Comment:
nit: perhaps this variable name doesn't make sense anymore, should be called `isBatch`?

Reply:
Changed to isWorkflow since it checks whether the programType is "workflows"